app/services/users/log_profile_update_activity_service.rb

```diff
@@ -0,0 +1,22 @@
+module Users
+  class LogProfileUpdateActivityService
```

Comment:
LogUsernameUpdateService sounds a bit more appropriate here, looking to what the service does.